emnify/assets/dashboards/emnify_dashboard.json

```
@@ -0,0 +1,459 @@
+{
+  "description": "Observe your consumption analytics all in one dashboard. Analyze patterns of usage across your devices inventory.",
+  "id": "ajg-fdu-uyn",
+  "is_read_only": false,
+  "layout_type": "ordered",
+  "notify_list": [ ],
+  "reflow_type": "fixed",
+  "template_variables": [ ],
+  "title": "[EMnify] Dashboard",
+  "widgets": [
+    {
+      "id": 973562992734448,
+      "definition": {
+        "title": "About EMnify Dashboard",
+        "type": "group",
+        "background_color": "vivid_orange",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 2719566117138984,
+            "definition": {
+              "type": "image",
+              "url": "https://portal.emnify.com/images/main-layout-logo-dark.svg",
+              "sizing": "contain",
+              "margin": "md",
+              "has_background": false,
+              "has_border": false,
+              "vertical_align": "center",
+              "horizontal_align": "center"
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 3,
+              "height": 2
+            }
+          },
+          {
+            "id": 4205652380459396,
+            "definition": {
+              "type": "note",
+              "content": "# Widgets\n\nObserve your consumption analytics all in one dashboard. Analyze usage patterns for your device inventory.\n\n# Setup\nConfigure streaming of Usage Data following the [integration guide][2],\nsections “Integration Steps” and “Verifying the integration”.\n\n# Dashboard Tuning\n\nYou can pick a period to filter the data displayed.\n\n[2]: https://www.emnify.com/integration-guides/emnify-datastreamer-integration-for-datadog",
+              "background_color": "white",
+              "font_size": "14",
+              "text_align": "left",
+              "vertical_align": "top",
+              "show_tick": false,
+              "tick_pos": "50%",
+              "tick_edge": "left",
+              "has_padding": true
+            },
+            "layout": {
+              "x": 0,
+              "y": 2,
+              "width": 6,
+              "height": 3
+            }
+          }
+        ]
+      },
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 6,
+        "height": 6
+      }
+    },
+    {
+      "id": 6446661413737418,
+      "definition": {
+        "title": "Total Data Consumption",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "query_value",
+        "requests": [
+          {
+            "formulas": [
+              {
+                "formula": "query1"
+              }
+            ],
+            "response_format": "scalar",
+            "queries": [
+              {
+                "query": "sum:endpoint.volume{*}.as_count()",
```

As mentioned, metrics from the integration should use a metric prefix, so we should change this to:

```suggestion
                "query": "sum:emnify.endpoint.volume{*}.as_count()",
```

as well as the rest of the dashboard and the monitors.